JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -155,14 +155,16 @@ module Microsoft.ApplicationInsights {
          * Starts timing how long the user views a page or other item. Call this when the page opens. 
          * This method doesn't send any telemetry. Call {@link stopTrackTelemetry} to log the page when it closes.
          * @param   name  A string that idenfities this item, unique within this HTML document. Defaults to the document title.
+         * @param StartDate A date that identifies the original start date that will override the current date. Defaults to the current date.
          */
-        public startTrackPage(name?: string) {
+        public startTrackPage(name?: string, StartDate?: Date) {
```

Comment:
`name` argument is optional. Please make sure this will work if a user didn't pass it and we automatically added startDate. 

Also, please change argument name to lower case `startDate`.